promxy.yaml

```
@@ -0,0 +1,45 @@
+package:
+  name: promxy
+  version: 0.0.90
+  epoch: 0
+  description: An aggregating proxy to enable HA prometheus.
+  copyright:
+    - license: MIT
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/jacksontj/promxy
+      tag: v${{package.version}}
+      expected-commit: 114942b664615bc68a67f56d2e458d7ca88f18d2
+
+  - uses: go/build
+    with:
+      packages: ./cmd/promxy
+      tags: netgo,builtinassets
+      output: promxy
+
+  - uses: go/build
+    with:
+      packages: ./cmd/remote_write_exporter
+      output: remote_write_exporter
+
+subpackages:
+  - name: promxy-compat
+    description: "Compatibility package to allow replacing promxy image"
```

we can write something on this line. 
```suggestion
    description: "Compatibility package to symlink binaries to match upstream deployment"
```

---

Thank you. Updated it via another commit.